contracts/veBalFeeInjector.sol

```diff
@@ -100,9 +106,8 @@ contract veBalFeeInjector is ConfirmedOwner, Pausable {
    * @notice Inject fees into veBAL distributor based on token balances and half, assuming it is past the end of the last epoch.
    *
    */
-  function _payFees() internal  whenNotPaused {
+  function _payFees() internal  {
```

Comment:
Consider removing the whenNotPaused modifier on _payFees() as the modifier has already been evaluated during payFees() or performUpkeep() or the other way around.